irohad/synchronizer/impl/synchronizer_impl.cpp

```diff
@@ -66,56 +66,32 @@ namespace iroha {
     }
 
     void SynchronizerImpl::processApplicableBlock(
-        const shared_model::interface::BlockVariant &committed_block_variant)
-        const {
-      iroha::visit_in_place(
-          committed_block_variant,
-          [&](std::shared_ptr<shared_model::interface::Block> block_ptr) {
-            auto storage = createTemporaryStorage();
-            if (not storage) {
-              return;
-            }
-            storage->apply(*block_ptr, trueStorageApplyPredicate);
-            mutable_factory_->commit(std::move(storage));
-
-            notifier_.get_subscriber().on_next(
-                SynchronizationEvent{rxcpp::observable<>::just(block_ptr),
-                                     SynchronizationOutcomeType::kCommit});
-          },
-          [this](std::shared_ptr<shared_model::interface::EmptyBlock>
-                     empty_block_ptr) {
-            notifier_.get_subscriber().on_next(SynchronizationEvent{
-                rxcpp::observable<>::empty<
-                    std::shared_ptr<shared_model::interface::Block>>(),
-                SynchronizationOutcomeType::kCommitEmpty});
-          });
+        std::shared_ptr<shared_model::interface::Block> commit_message) const {
```

Comment:
Why did you name parameter so differently? I think, it's much better to call it `committed_block` instead of some message

Reply:
In general it is commit_message, not block. It might be either block or reject case in future